app/support/reservations/validations.rb

```diff
@@ -18,7 +18,7 @@ module Reservations::Validations
     end
 
     validate :starts_before_ends
-    validate :duration_is_interval, on: :create
+    validate :duration_is_interval
```

Comment:
Can you test this out from the order detail modal (admin side) and make sure it displays a nice error? I'm not sure we were enforcing this on that side before. Also, make sure this works for Create & Start where I think it pulls the start time forward a little bit.